src/box/lua/schema.lua

```diff
@@ -2340,8 +2340,21 @@ end
 
 box.internal.check_select_opts = check_select_opts -- for net.box
 
+local check_select_args_rl = log.internal.ratelimit:new()
```

Comment:
For me, it would be better if the name of the variable described what it is for: something like `dangerous_select_logger`, for example.

Reply:
I want this object to be _tied_ to the `check_select_args` function and provide its state (just like a static function variable would in _C_) — hence the naming.